src/subdomains/generic/user/models/user/dto/linked-user.dto.ts

```diff
@@ -9,12 +9,18 @@ export class LinkedUserInDto {
   @IsString()
   @Matches(GetConfig().addressFormat)
   address: string;
+
+  @ApiPropertyOptional()
+  @IsOptional()
+  @IsEnum(Blockchain)
+  blockchain: Blockchain;
```

Comment:
In this case we can make it required. This functionality is only used by payment page.